README.rst

```diff
@@ -21,12 +21,47 @@ Setup
 ~~~~~
 
 Install python dependencies for acceptance testing (on a Jenkins instance
-running within a Docker container).
+running within a Docker container), as well as various Groovy helper utilities.
 
 .. code:: bash
 
 make requirements
```

Comment:
I know this was already merged, but this should be indented so it appears as code